### ISSUE TITLE
Fullscreen with zoomed font

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#30]
```
Fullscreen mode in MinTTY fills up the whole screen with as many rows and
columns as the current font size allows, thus maximising the space
available, much like e.g. in web browsers.

Fullscreen mode in the DOS box on the other hand (when it was still
supported), switched the graphics card into character mode and kept the
number of rows and columns, thereby in effect zooming the window contents
up to the whole screen.

Character mode is out of the question for MinTTY, but something similar
could be done by scaling up the font size. Since the window geometry is
unlikely to exactly fit the screen geometry, the font size could be
increased as far as possible while still fitting all existing rows and columns.

Any remaining space could be filled up with additional character cells, but
those would have to be cut away again when going back to window mode.
```

Original issue reported on code.google.com by `andy.koppe` on 8 Jan 2009 at 6:05

**Comment by mintty** [comment]
This is meanwhile available, see #204.

**Comment by mintty** [comment]
General rule is now: when holding Shift while resizing or zooming, window size and font size are kept in sync.
